src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -128,7 +128,8 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         virtual
         returns (uint mintAmount)
     {
-        _validateDepositAmount(_depositAmount);
```

Comment:
One thing to ensure amount really never is 0 would be to check also in _calculateNetAndSplitFees that the total fees are not greater than or equal to BPS (it currently allows it to be equal which will result in amount 0)

Reply:
Ok, I don't see any issue to leave that, but then I think it'll jjust be more accurate to make it < instead of <= since it is never meant to be equal and will be better to fail early.

Reply:
Makes sense, @Zitzak addresses this [here](https://github.com/InverterNetwork/contracts/pull/619/commits/d4c37d1932543f6e36fc2c8609ace6a240355a03).